exposed-json/src/test/kotlin/org/jetbrains/exposed/sql/json/JsonBColumnTests.kt

```diff
@@ -197,8 +197,9 @@ class JsonBColumnTests : DatabaseTestsBase() {
             assertEquals(singleId, tester.selectAll().where { firstIsOnTeamA }.single()[tester.id])
 
             // older MySQL and MariaDB versions require non-scalar extracted value from JSON Array
+            val toScalar = testDb != TestDB.MYSQL_V5
             val path2 = if (currentDialectTest is PostgreSQLDialect) "0" else "[0]"
-            val firstNumber = tester.numbers.extract<Int>(path2, toScalar = !isOldMySql())
+            val firstNumber = tester.numbers.extract<Int>(path2, toScalar = toScalar)
```

Comment:
`binaryJsonNotSupportedDB` now includes `TestDB.MYSQL_V5` (should it?). That would make `toScalar` always true, no?

Reply:
Thank you, it was a wrong exclusion of MySql5 by default, it was necessary to exclude it for the test `testJsonBWithDefaults`. Now `toScalar == false` for MySql5 makes sense again)